### PR TITLE
Added option to add phone number to create and update user

### PR DESCRIPTION
**Pull request by gkhunger**
In our project we needed to use a user's phone number information.

### CODE REVIEW

**Comment by nebhale** [comment]
@gkhunger Let me know when you've signed the CLA so that we can get this merged in quickly.

**Comment by gkhunger** [comment]
Do I need to be a cloud foundry member to sign it?

**Comment by nebhale** [comment]
@gkhunger You do not.  You can just follow the links there and the instructions will guide you along.

**Comment by gkhunger** [comment]
Somehow the link takes me to https://www.cloudfoundry.org/governance/

**Comment by nebhale** [comment]
@chipchilders It appears that something has broken the CLA links on Cloud Foundry website.  Tapping the [Individual CLA link](https://www.cloudfoundry.org/governance/cff_individual_cla/) results in a loop back to the main governance page.  Can you please have someone look into that?

**Comment by chipchilders** [comment]
Yes - getting our web team to look right now.

File: cloudfoundry-client/src/main/java/org/cloudfoundry/uaa/users/_CreateUserRequest.java [hunk 1]
```diff
@@ -92,4 +92,11 @@ void check() {
     @Nullable
     abstract Boolean getVerified();
 
+    /**
+     * The phone number
```
Comment (twoseat) [inline cloudfoundry-client/src/main/java/org/cloudfoundry/uaa/users/_CreateUserRequest.java]: Should be plural

File: cloudfoundry-client/src/main/java/org/cloudfoundry/uaa/users/_PhoneNumber.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ */
+
+package org.cloudfoundry.uaa.users;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.immutables.value.Value;
+
+/**
+ * The email addresses for a user
```
Comment (twoseat) [inline cloudfoundry-client/src/main/java/org/cloudfoundry/uaa/users/_PhoneNumber.java]: Copy/paste error (my speciality!)

File: cloudfoundry-client/src/main/java/org/cloudfoundry/uaa/users/_UpdateUserRequest.java [hunk 3]
```diff
@@ -102,4 +102,10 @@ void check() {
     @Nullable
     abstract Boolean getVerified();
 
+    /**
+     * The phone numbers for the user
+     */
+    @JsonProperty("phoneNumbers")
+    abstract List<PhoneNumber> getPhoneNumbers();
```
Comment (twoseat) [inline cloudfoundry-client/src/main/java/org/cloudfoundry/uaa/users/_UpdateUserRequest.java]: Fields should be in alphabetical order (alt-cmd-L will fix that)

File: cloudfoundry-client/src/main/java/org/cloudfoundry/uaa/users/AbstractUserSummary.java [hunk 4]
```diff
@@ -88,4 +88,10 @@
     @JsonProperty("zoneId")
     public abstract String getZoneId();
 
+    /**
+     * The phone numbers for the user
+     */
+    @JsonProperty("phoneNumbers")
+    public abstract List<PhoneNumber> getPhoneNumbers();
```
Comment (twoseat) [inline cloudfoundry-client/src/main/java/org/cloudfoundry/uaa/users/AbstractUserSummary.java]: Fields should be in alphabetical order

**Comment by chipchilders** [comment]
@gkhunger @nebhale sorry about the CLA link issues. The team resolve it just now. Try again!

**Comment by nebhale** [comment]
@chipchilders Looks good from my side.

**Comment by gkhunger** [comment]
Works for me too. Thanks.

**Comment by gkhunger** [comment]
I received an email from Chris Clark that he received my CLA. Can we merge the code now?

**Comment by nebhale** [comment]
Let's find out...

**Comment by nebhale** [comment]
Looks good.

**Pull request closed without merging**